examples/Thermostat/slice/ThermoControl.slice

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Controls the actual thermostat.
+interface ThermoControl {
+    /// Sets a new target temperature (known as set point).
+    /// @param setPoint: The new target temperature.
```

Comment:
Should we specify this is `(in °F)`? We seem to mention that fact everywhere else we talk about the temperature.

Reply:
Fixed